website/docs/artist_getting_started.md

```diff
@@ -18,13 +18,17 @@ If this is not the case, please contact your administrator to consult on how to
 If you are working from home though, you'll need to install it yourself. You should, however, receive the OpenPype installer files from your studio
 admin, supervisor or production, because OpenPype versions and executables might not be compatible between studios.  
 
-To install OpenPype you just need to unzip it anywhere on the disk
+Installing OpenPype is possible by Windows installer or by unzipping it anywhere on the disk from downloaded ZIP archive.
 
-To use it, you have two options
+There are two options running OpenPype
 
-**openpype_gui.exe** is the most common for artists. It runs OpenPype GUI in system tray. From there you can run all the available tools. To use any of the features, OpenPype must be running in the tray.
+first most common one by using OP icon on the Desktop triggering
```

Comment:
Admittedly this is hard for me to follow...

I have two options to install:
- Installer
- Unzipping ZIP archive

I have two options to run, but the added  'first most common one' mention makes it hard to follow because it's not visually connected to the line that it should be connected to.
First off, I think when unzipping the ZIP archive, there won't be an OP icon on the desktop.
Second, I'd refrain from abbreviating OpenPype to OP early on in the getting started guide, just to remain very explicit to the end user.

Reply:
@BigRoy Yeah, you are right, good points! Sry just starting with writing docs being noobish! Thanks for your input will take it on my mind when correcting it!